vita3k/app/src/app_init.cpp

```diff
@@ -139,7 +139,12 @@ bool init(HostState &state, Config &cfg, const Root &root_paths) {
         break;
     }
 
-    state.window = WindowPtr(SDL_CreateWindow(window_title, SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED, DEFAULT_RES_WIDTH, DEFAULT_RES_HEIGHT, window_type | SDL_WINDOW_RESIZABLE), SDL_DestroyWindow);
+    if (cfg.fullscreen) {
+        state.display.fullscreen = true;
+        state.window = WindowPtr(SDL_CreateWindow(window_title, SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED, DEFAULT_RES_WIDTH, DEFAULT_RES_HEIGHT, window_type | SDL_WINDOW_FULLSCREEN_DESKTOP | SDL_WINDOW_RESIZABLE), SDL_DestroyWindow);
+    } else {
+        state.window = WindowPtr(SDL_CreateWindow(window_title, SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED, DEFAULT_RES_WIDTH, DEFAULT_RES_HEIGHT, window_type | SDL_WINDOW_RESIZABLE), SDL_DestroyWindow);
+    }
     if (!state.window || !init(state.mem) || !init(state.audio, resume_thread) || !init(state.io, state.base_path, state.pref_path)) {
```

Comment:
Just a suggestion, since the flags are the only thing that actually change between the two branches, maybe make flags a variable and pass it to SDL?

Reply:
what do you mean pass it to SDL?

Reply:
no need for a flag var even, just do `window_type |= SDL_WINDOW_FULLSCREEN_DESKTOP `

Reply:
i can't simply just do `window_type |= SDL_WINDOW_FULLSCREEN_DESKTOP` since window_type is an enum and SDL_WINDOW_FULLSCREEN_DESKTOP is an int, i had to cast them

Reply:
make window_type an int